### ISSUE TITLE
Fix pre-commit errors

### ISSUE THREAD

**Issue by max3903** [ursais/osi-addons#406]
```
account_bank_statement_extend/report/account_bank_statement.py:29:29: B007 Loop control variable 'move' not used within the loop body. If this is intended, start the name with an underscore.
account_bank_statement_extend/report/account_bank_statement.py:37:37: E741 ambiguous variable name 'l'
account_bank_statement_extend/report/account_bank_statement.py:50:49: F821 undefined name 'b_state'
account_bank_statement_extend/report/account_bank_statement.py:99:104: B950 line too long (103 > 88 characters)
account_bank_statement_extend/report/account_bank_statement.py:116:29: B007 Loop control variable 'move' not used within the loop body. If this is intended, start the name with an underscore.
account_bank_statement_extend/report/account_bank_statement.py:123:37: E741 ambiguous variable name 'l'
account_bank_statement_extend/report/account_bank_statement.py:177:104: B950 line too long (103 > 88 characters)
account_bank_statement_extend/wizard/parser_report_bank_statement.py:46:39: E741 ambiguous variable name 'l'
account_bank_statement_extend/wizard/parser_report_bank_statement.py:61:5: C901 'BankStatementReport.get_bank_statement_report_data' is too complex (23)
account_bank_statement_extend/wizard/parser_report_bank_statement.py:97:29: B007 Loop control variable 'move' not used within the loop body. If this is intended, start the name with an underscore.
account_bank_statement_extend/wizard/parser_report_bank_statement.py:104:37: E741 ambiguous variable name 'l'
account_bank_statement_extend/wizard/parser_report_bank_statement.py:163:29: B007 Loop control variable 'move' not used within the loop body. If this is intended, start the name with an underscore.
account_bank_statement_extend/wizard/parser_report_bank_statement.py:170:37: E741 ambiguous variable name 'l'
fieldservice_mobile/models/ir_attachment.py:37:36: B007 Loop control variable 'create_uid' not used within the loop body. If this is intended, start the name with an underscore.
```

**Comment by max3903** [comment]
CC: @bodedra

**Comment by max3903** [comment]
@Hardik-OSI Can you fix thoses errors? #407 is fixed.

**Comment by dreispt** [comment]
Related: #441 and #442
